### DISCUSSION TITLE
How to get results for all categories (e.g., wood, bottle, leather) at the same time?

### DISCUSSION THREAD

**Discussion by code10086web** [open-edge-platform/anomalib#522] [Q&A] [answered]

**Comment by djdameln** [comment, marked as answer]
Hi, this can be achieved with the benchmarking tool, as explained in [this guide](https://openvinotoolkit.github.io/anomalib/guides/benchmarking.html). To collect results across different dataset categories, simply list all categories in the `grid_search` section of your benchmarking config file and follow the instructions to run the benchmarking session.

**Reply by IamSVP94** [reply]
guide is not available now(

**Reply by ashwinvaidya17** [reply]
https://openvinotoolkit.github.io/anomalib/tutorials/benchmarking.html
